cluster/addons/fluentd-elasticsearch/es-statefulset.yaml

```diff
@@ -38,13 +40,19 @@ spec:
           name: transport
           protocol: TCP
         volumeMounts:
-        - name: es-persistent-storage
+        - name: elasticsearch-logging
           mountPath: /data
         env:
         - name: "NAMESPACE"
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
-      volumes:
-      - name: es-persistent-storage
-        emptyDir: {}
+  volumeClaimTemplates:
+  - metadata:
+      name: elasticsearch-logging
+    spec:
+      storageClassName: gp2
```

Comment:
this isn't portable, its AWS specific. allow this to be the cluster default (which I think is an empty string?  the semantics of storage class default is very particular).